sdk/template/template/package.json

```diff
@@ -22,19 +22,16 @@
     "execute:samples": "echo skipped",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
-    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts -f html -o template-lintReport.html || exit 0",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
-    "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha --reporter ../../../common/tools/mocha-multi-reporter.js dist-test/index.node.js",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+    "test": "npm run build:test && npm run test:node && npm run test:browser",
+    "test-ci": "npm run build:test && npm run test-ci:node && npm run test-ci:browser",
+    "test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-test/index.node.js",
```

Comment:
The `test:node` command helped with a convenience that developers don't have to run `clean` and `build:test` commands before running the tests each time.
This is taking away that convenience. 
The same applies to `test:browser`.

Reply:
I have added this back! This is how it looks now:

```
  "test": "npm run clean && npm run build:test && npm run test:node && npm run test:browser",
  "test-ci": "npm run clean && npm run build:test && npm run test-ci:node && npm run test-ci:browser",
```

Thank you for reminding me about this.

Reply:
This is still taking away the convenience of not allowing executing only the node tests with "clean" and "build:test".

Reply:
Thanks! I've moved them to this:

```json
  "test": "npm run test:node && npm run test:browser",
  "test-ci": "npm run test-ci:node && npm run test-ci:browser",

  // Where the individual commands are:
  "test:node": "npm run clean && npm run build:test && mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-test/index.node.js",
  "test-ci:node": "npm run clean && npm run build:test && nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-esm/test/**/*.spec.js",
  // Remember to specify different karma.config.js if needed
  "test:browser": "npm run clean && npm run build:test && karma start --single-run",
  "test-ci:browser": "npm run clean && npm run build:test && npm run test:browser",

  // Where we're cleaning with:
  "clean": "rimraf dist dist-* types",
```

Reply:
One small correction.. 
>   "test-ci:browser": "npm run clean && npm run build:test && npm run test:browser",

to 

>  "test-ci:browser": "npm run test:browser",

----------------------------------

Our pipelines have separate `build`, `build:test`, `integration-test` steps as part of the job.
With your current proposal, those steps are being duplicated.

Reply:
Regarding the first point:

Sure! I can do that.

Regarding the second point:

Yep, we have to follow up after this PR fixing a bunch of things.